orc8r/cloud/go/services/configurator/storage/storage.proto

```diff
@@ -208,11 +208,20 @@ message EntityLoadCriteria {
     bool load_assocs_from_this = 4;
 
     bool load_permissions = 5;
+
+    // The following parameters allow for pagination of entity loads.
+    // Size of page to return
+    uint64 page_size = 6;
+    // Token for which all returned keys will be greater than
+    string page_token = 7;
```

Comment:
Both these page tokens should be the base64-encoded serialized value of an `EntityPageToken` proto message, defined in this file. That gives us stronger flexibility in forward compatibility

Since this page token will get passed all the way out through the REST API, doing things this way also adds the affordance of "this is an opaque page token*